supervised-model-quality/readme.md

```diff
@@ -0,0 +1,18 @@
+# Supervised model quality
+
+In this package you'll find a script that generates an evaluation from
+an existing model by retrieving the dataset used for training and doing
+an 80%-20% split of the data. The output of the script is a map with the
```

Comment:
If I understood well, we are retraining the model with a subset of the original data and testing it with the other subset, aren't we?  Maybe the user can think that we are evaluating the same model with a subset of the training data...

Anyways, I am testing it and I am having some problems:


ERROR: Problem while executing script: Function expects 7 arguments, 5 given

PROBLEM FOUND:
	with push-procedure at line 29 column 24

STACK TRACE:
	at line 29, column 24
	at line 29, column 25
	at line 51, column 21

Reply:
This problem should be solved now. The function I used to retrieve the model configuration had changed and new arguments were needed. Thanks for the heads up.